home/style.css

```diff
@@ -0,0 +1,162 @@
+@import url("../css/font.css");
+@import url("../css/nav.css");
+html {
+  width: 100%;
+  font-size: 8px;
+  font-family: "Noto Sans";
+}
+body {
+  height: 100vh;
+  border: 0;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  position: relative;
+}
+
+main {
+  height: 90vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+#main__container {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  position: relative;
+  overflow: auto;
+  position: sticky;
+  
+}
+#main__container::-webkit-scrollbar {
+  display: none;
+}
```

Comment:
Avoid grabbing elements by their IDs, try using classes instead.

Reply:
Sure Will try to fix that. Thank You for reviewing it!